rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
+import { terser } from 'rollup-plugin-terser';
+
+export default {
+  input: 'lib/ical/module.js',
+  output: [
+    { file: 'dist/ical.js',  format: 'es', exports: "default" },
```

Comment:
package.json contains `"main": "build/ical.js`, while the build scripts puts the compiled files under `dist/`.  I am sceptical whether this will work.

Reply:
Taken care of, and another reference to `build`. Thanks!